<?xml version="1.0"?>
<launch>

  <node ns="left_arm_ft_sensor" name="drift_compensation" pkg="dumbo_ft_drift_compensation" type="ft_drift_compensation_node" cwd="node" respawn="false" output="screen">
    <rosparam command="load" file="$(find dumbo_ft_drift_compensation)/config/dumbo_left_arm_ft_drift_compensation.yaml" />
    <remap from="/left_arm_ft_sensor/drift_compensation/ft_drift_compensated" to="/left_arm_ft_sensor/ft_drift_compensated" />
    <remap from="/left_arm_ft_sensor/drift_compensation/ft" to="/left_arm_ft_sensor/ft_gravity_compensated" />
    <remap from="/left_arm_ft_sensor/drift_compensation/imu" to="/imu/data" />
    <remap from="/left_arm_ft_sensor/trajectory_execution_event" to="/trajectory_execution_event" />
    <remap from="/left_arm_ft_sensor/attached_collision_object" to="/attached_collision_object" />
    <remap from="/left_arm_ft_sensor/move_group/cancel" to="/move_group/cancel" />
    <remap from="/left_arm_ft_sensor/move_group/feedback" to="/move_group/feedback" />
    <remap from="/left_arm_ft_sensor/move_group/result" to="/move_group/result" />
    <remap from="/left_arm_ft_sensor/move_group/status" to="/move_group/status" />
    <remap from="/left_arm_ft_sensor/move_group/goal" to="/move_group/goal" />
    <remap from="/left_arm_ft_sensor/pickup/cancel" to="/pickup/cancel" />
    <remap from="/left_arm_ft_sensor/pickup/feedback" to="/pickup/feedback" />
    <remap from="/left_arm_ft_sensor/pickup/result" to="/pickup/result" />
    <remap from="/left_arm_ft_sensor/pickup/status" to="/pickup/status" />
    <remap from="/left_arm_ft_sensor/pickup/goal" to="/pickup/goal" />
    <remap from="/left_arm_ft_sensor/place/cancel" to="/place/cancel" />
    <remap from="/left_arm_ft_sensor/place/feedback" to="/place/feedback" />
    <remap from="/left_arm_ft_sensor/place/result" to="/place/result" />
    <remap from="/left_arm_ft_sensor/place/status" to="/place/status" />
    <remap from="/left_arm_ft_sensor/place/goal" to="/place/goal" />
  </node>

</launch>